nodewatcher/modules/monitor/http/survey/management/commands/export_survey_data.py

```diff
@@ -0,0 +1,81 @@
+import io
+import json
+import time
+
+from django.core.management import base
+
+from django_datastream import datastream
+
+
+class Command(base.BaseCommand):
+    help = 'Consolidates survey data from all nodes and exports the meta graph as a JSON file.'
+
+    def add_arguments(self, parser):
+
+        # Optional arguments
+        parser.add_argument(
+            '--surveys-to-export',
+            type=int,
+            action='store',
+            dest='number',
+            default=1,
+            help='Number of surveys to export'
+        )
+
+        parser.add_argument(
+            '--label',
+            type=str,
+            action='store',
+            dest='label',
+            help='Optional label for the filename',
+        )
+
+    def handle(self, *args, **options):
+        """
+        Exports the latest survey data graph as a JSON file into the root directory.
+        """
+
+        number = options['number']
+        label = options['label']
+
+        streams = datastream.find_streams({'module': 'monitor.http.survey'})
+
+        # Create a meta graph
+        meta_vertices, meta_edges = [], []
+        friendly_nodes = []
+        for stream in streams:
+            datapoint_iterator = datastream.get_data(
+                stream_id=stream['stream_id'],
+                granularity=stream['highest_granularity'],
+                start=stream['latest_datapoint'],
+                reverse=True,
+            )
+            try:
+                stream_graph = datapoint_iterator[0]['v']
+                for vertex in stream_graph['v']:
+                    meta_vertices.append(vertex)
+                    if 'bssids' in vertex:
+                        friendly_nodes.append(vertex)
+                for edge in stream_graph['e']:
+                    meta_edges.append(edge)
+            except IndexError:
+                pass
+        meta_graph = {
+            'v': meta_vertices,
+            'e': meta_edges,
+        }
+        exported_graph = {
+            'graph': meta_graph,
+            'friendly_nodes': friendly_nodes,
+        }
+
+        timestamp = int(time.time())
+
+        if label:
+            filename = label + str(timestamp) + '.json'
+        else:
+            filename = 'survey_export' + str(timestamp) + '.json'
+
+        with io.open(filename, 'w', encoding='utf-8') as f:
```

Comment:
It is pretty common to allow user to simply specify filename/path as an argument and this is it. You want also to specify the timestamp into the filename and this makes it more complicated. Is timestamp really necessary?

I would maybe make it so that you have one argument which is the timestamp for which you want to get data, and if it is not specified, current time is used. So, user/caller knows the timestamp (or it is given, or it is current time) and can use Bash or something else to construct the filename if they care about having the timestamp in the filename.

So maybe simplify this so that simply it is just a filename.